### PR TITLE
psd 912 asb pod identity

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-912

### CODE REVIEW

File: helm/ffc-demo-payment-service/templates/container-secret.yaml [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
   B2C_CLIENT_ID: {{ .Values.b2c.clientId | quote }}
+  B2C_CLIENT_SECRET: {{ .Values.b2c.clientSecret | quote }}
   {{- end }}
   {{- if .Values.okta.domain }}
-  OKTA_CLIENT_ID: {{ .Values.okta.clientId | quote }}
   OKTA_AUTH_SERVER_ID: {{ .Values.okta.authorizationServerId | quote }}
+  OKTA_CLIENT_ID: {{ .Values.okta.clientId | quote }}
   {{- end }}
-  APPINSIGHTS_INSTRUMENTATIONKEY: {{ quote .Values.appInsights.key }}
+  POSTGRES_PASSWORD: {{ .Values.postgresService.postgresPassword | quote }}
```
Comment (paulsimonandrews) [inline helm/ffc-demo-payment-service/templates/container-secret.yaml:13]: Are you also going to integrate the managed identity postgres changes into this PR too?

File: server/services/messaging/message-base.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 const { ServiceBusClient } = require('@azure/service-bus')
 
 class MessageBase {
-  constructor (name, config) {
+  constructor (name, config, credentials, action) {
     this.name = name
     const connectionString = `Endpoint=sb://${config.host}/;SharedAccessKeyName=${config.username};SharedAccessKey=${config.password}`
     this.sbClient = ServiceBusClient.createFromConnectionString(connectionString)
```
Comment (paulsimonandrews) [inline server/services/messaging/message-base.js:7]: Missing the code to create the `sbClient` from the token provider if credentials object is not `undefined`

File: helm/ffc-demo-payment-service/templates/container-secret.yaml (hunk 1 above)
Comment (st3v3nhunt) [reply helm/ffc-demo-payment-service/templates/container-secret.yaml:13]: I was going to do it as a separate PR for ease of review and keeping it focused.

File: server/services/messaging/message-base.js (hunk 2 above)
Comment (st3v3nhunt) [reply server/services/messaging/message-base.js]: Thanks, great spot. Changes pushed.

**Pull request merged**